Dockerfile

```diff
@@ -2,16 +2,22 @@
 #
 # VERSION	0.1
 
-FROM debian:jessie-slim
-MAINTAINER Richard North <rich.north@gmail.com>
+FROM debian:10.5-slim
 
 LABEL Description="This image can be used to create a sidekick container for recording videos of VNC sessions hosted in other containers"
+LABEL maintainer="Richard North <rich.north@gmail.com>"
 
 RUN apt-get update && apt-get install -y \
   python-pip python-dev \
-  && rm -rf /var/lib/apt/lists/*
+  && rm -rf /var/lib/apt/lists/* \
+  && pip install vnc2flv \
+  && rm -fr /tmp/*
 
-RUN pip install vnc2flv
+# INSTALL ffmpeg
+RUN apt-get update && \
+    apt-get install -y ffmpeg=7:4.1.6-1~deb10u1 && \    
+    apt-get autoremove && apt-get clean && \
+    ffmpeg -version
```

Comment:
Sorry, one more thing to change:
Can you please move all those commands to the `RUN` command starting line 10?

Line10 should read like this I assume:
```
RUN apt-get update && apt-get install -y \
  python-pip python-dev ffmpeg=7:4.1.6-1~deb10u1 \	 
  && rm -rf /var/lib/apt/lists/*
  && pip install vnc2flv \
  && rm -fr /tmp/*
```

This way we have minimal layers. 

Is `autoremove` and `clean` necessary? There should be no useless packages coming in I would have assumed.

Reply:
Done